internal/keymagic/keymagic_test.go

```diff
@@ -6,10 +6,11 @@ import (
 	"github.com/bouk/monkey"
 	"github.com/google/uuid"
 	config "github.com/quan-to/chevron/internal/config"
-	"github.com/quan-to/chevron/internal/etc"
 	"github.com/quan-to/chevron/internal/keybackend"
+	"github.com/quan-to/chevron/internal/tools"
 	"github.com/quan-to/chevron/internal/vaultManager"
 	"github.com/quan-to/chevron/pkg/QuantoError"
+	"github.com/quan-to/chevron/pkg/database/memory"
 	"github.com/quan-to/chevron/pkg/interfaces"
 	"github.com/quan-to/chevron/test"
 	"os"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).